docs/reference/docs/reindex.asciidoc

```diff
@@ -601,3 +601,29 @@ and it'll look like:
 // TESTRESPONSE
 
 Or you can search by `tag` or whatever you want.
+
+
+[float]
+=== Reindex daily indices
+
+You can use `_reindex` in combination with painless to reindex daily indices to apply a new template.
+Painless extracts the date of the index name and creates a new index with `-1` appended. All data 
+from `metricbeat-2016.05.31` will be reindex into `metricbeat-2016.05.31-1`.
+
```

Comment:
The advantage of something like this is that `gradle docs:check` will actually test the thing that you are telling the user to do so if it breaks we'll know.